NLibrary/Classes/PlaylistMVVM/PlaylistItemBasicModel.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  UserProtocol.swift
+//  taylorswift
+//
+//  Created by Nathan Ngobale on 2020/03/20.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import Foundation
+import AVKit
+
+public struct PlaylistBasicModel {
+    public var playlistItems: [PlaylistItemBasicModel] = []
+    public var authorId: String?
+    init?(dict: NSDictionary) {
+        guard let keyValues = dict.allValues.first as? NSDictionary else {
+            return
+        }
+        guard let authorId = dict.allKeys.first as? String else {
+            return
+        }
+        self.authorId = authorId
+        for (key, value) in keyValues {
+            guard let key = key as? String,
+                let page = value as? NSDictionary else {
+                return
+            }
+            playlistItems.append(PlaylistItemBasicModel(albumName: key, dictionary: page))
+        }
+    }
+    
+    var dict: [String : Any]?
+}
+
+public struct PlaylistItemBasicModel: DictionaryConvertible {
+    public var id: String = ""
+    public var name: String = ""
+    public var numSongs: Int = 0
+    public var basicSongItems: [BasicSongItem] = []
+    public var dict: [String: Any] {
+        return [
+            "name": name,
+            "numSongs": numSongs
+        ]
+    }
+
+    init() {
+    }
+
+    init?(dict: [String: Any]) {
```

Comment:
Why are you passing a parameter if you're not using it? The self.init() doesn't have code making this enture initializer seem redundant.